### PR TITLE
Fixing and regenerating swagger documentation

### PR DESCRIPTION
**Pull request by jespino**

### CODE REVIEW

File: server/main/doc.go [hunk 1]
```diff
@@ -0,0 +1,26 @@
+// Package classification Focalboard Server
```
Comment (jespino) [inline server/main/doc.go:1]: Moving the general documentation to an independent file fixes the home path generated reference.

File: server/api/api.go [hunk 2]
```diff
@@ -1635,7 +1635,7 @@ func (a *API) handleDeleteSubscription(w http.ResponseWriter, r *http.Request) {
 	//   description: Workspace ID
 	//   required: true
 	//   type: string
-	// - name: BlockID
+	// - name: blockID
```
Comment (jespino) [inline server/api/api.go:1638]: Not related issue, that I had to fix to generate the swagger documentation.

**Comment by jespino** [comment]
Almost everything is generated code, the only changes are in the `server/api/api.go`, in `server/main/doc.go` and in `server/main/main.go` files.

**Review by wiggin77 — APPROVED** [review]
LGTM :+1:    Thanks for fixing `BlockID`

File: server/api/api.go (hunk 2 above)
Comment (wiggin77) [reply server/api/api.go:1638]: I forgot to re-generate the the swagger files when I checked that in...  otherwise I would have caught that.

Comment (jespino) [reply server/api/api.go:1638]: Yes, maybe the CI should check that, just "generate and see if something changes or not"

**Review by sbishel — APPROVED** [review]
LGTM - we should probably discuss swagger at a dev meeting to make sure everyone understands how to use and rebuild.

**Pull request merged**